collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java

```diff
@@ -96,6 +96,7 @@ public void playAudio() {
             // (it's a spanned thing...)
             viewText.setText(viewText.getText().toString());
             viewText.setTextColor(playTextColor);
+            viewText.setBackgroundColor(playBackgroundTextColor);
```

Comment:
@lognaturel `playBackgroundTextColor` was not being used anywhere so I have set it here because it looks like it got missed by mistake